### DISCUSSION TITLE
Are grpc-spring-boot-starter v4.9.1 support springboot 2.7.11 and java 17 ?

### DISCUSSION THREAD

**Discussion by kridtakom** [LogNet/grpc-spring-boot-starter#354] [Q&A]
Now I using springboot 2.7.11 and java 17 for my project. Can you tell me, grpc-spring-boot-starter v4.9.1 is compattible with my project?

Thank you so much.

**Comment by jvmlet** [comment]
yes, should be. Are you getting any errors ?
